frontend/pluginloader.lua

```diff
@@ -104,66 +126,93 @@ function PluginLoader:loadPlugins()
             table.insert(lookup_path_list, extra_path)
         end
     end
-
-    -- keep reference to old value so they can be restored later
-    local package_path = package.path
-    local package_cpath = package.cpath
-
-    local plugins_disabled = G_reader_settings:readSetting("plugins_disabled")
-    if type(plugins_disabled) ~= "table" then
-        plugins_disabled = {}
-    end
-    -- disable obsolete plugins
-    for element in pairs(OBSOLETE_PLUGINS) do
-        plugins_disabled[element] = true
-    end
     for _, lookup_path in ipairs(lookup_path_list) do
-        logger.info("Loading plugins from directory:", lookup_path)
+        logger.info("Discovering plugins from directory:", lookup_path)
```

Comment:
Search for, Looking for ?

Reply:
Both of those are fine (and better than discovering).